plugins/woocommerce/patterns/coming-soon-entire-site.php

```diff
@@ -156,15 +173,12 @@
 body .is-layout-constrained > .woocommerce-coming-soon-banner.alignwide {
 	max-width: 820px;
 }
-.coming-soon-is-vertically-aligned-center:not(.block-editor-block-list__block) {
-	position: absolute;
-	top: 50%;
-	transform: translateY(-50%);
-	margin-block-start: 0;
+.coming-soon-is-vertically-aligned-center {
 	width: 100%;
+	align-items: stretch;
 }
-.woocommerce-coming-soon-header, .coming-soon-cover .wp-block-cover__background {
-	background-color: #bea0f2 !important;
+.woocommerce-coming-soon-header {
+	height: 40px;
```

Comment:
Small hack: Fixed header height in order to ensure small modifications will keep the center part relatively stable